contracts/sol6/IKyberFeeHandler.sol

```diff
@@ -1,11 +1,13 @@
 pragma solidity 0.6.6;
 
+import "./IERC20.sol";
+
 
 interface IKyberFeeHandler {
-    event RewardPaid(address indexed staker, uint256 indexed epoch, uint256 amountWei);
-    event RebatePaid(address indexed rebateWallet, uint256 amountWei);
-    event PlatformFeePaid(address indexed platformWallet, uint256 amountWei);
-    event KncBurned(uint256 kncTWei, uint256 amountWei);
+    event RewardPaid(address indexed staker, uint256 indexed epoch, IERC20 indexed token, uint256 amount);
+    event RebatePaid(address indexed rebateWallet, IERC20 indexed token, uint256 amount);
+    event PlatformFeePaid(address indexed platformWallet, IERC20 indexed token, uint256 amount);
+    event KncBurned(uint256 kncTWei, IERC20 indexed token, uint256 amount);
 
     function handleFees(
```

Comment:
Maybe `platformFeeWei` -> `platformFeeTwei`?

Reply:
Also, `uint` -> `uint256` for the burnAmountWei

Reply:
If we do `platformFeeWei -> platformFeeTwei`, then should do for all the other params as well.. like `rewardWei`, `rebateWei`, `burnAmountWei`....

Reply:
The events are designed for fee in ETH + tokens, but current FeeHandler is designed only for fee in ETH, I think the params in current FeeHandler should be still `wei` amount.

Reply:
yeah ok